### DISCUSSION TITLE
Public policy doesn't show restricted role count

### DISCUSSION THREAD

**Discussion by w0kyj** [directus/directus#24943] [Feature Requests]
### Describe the Bug

Assigning a policy to the `Public` restricted role doesn't show in the policy role counts or as a role relationship inside the policy

### To Reproduce

1. Create a "public" policy.
1. Update the `Public` role to include this policy
1. Return to policies and note the role count is still 0
1. Open the "public" policy and note the `Public` role is not shown
<img width="456" alt="Screenshot 2024-08-08 at 3 59 07 PM" src="https://github.com/user-attachments/assets/2ee0281d-0920-4d36-adf1-3d7da760ba27">
<img width="491" alt="Screenshot 2024-08-08 at 3 58 57 PM" src="https://github.com/user-attachments/assets/43aef1f1-fceb-4c67-8add-6fd41bbfa292">
<img width="792" alt="Screenshot 2024-08-08 at 3 58 46 PM" src="https://github.com/user-attachments/assets/13a4671d-49e4-4097-b8d4-a0247ddbe8bd">

### Directus Version

v11.0.1

### Hosting Strategy

Directus Cloud

### Database

PostgreSQL

**Comment by pomeh** [comment]
Hello,

I can confirm this behavior (Directus v11.0.2 / Self-Hosted (Docker Image) / SQLite).

As far as I understand, this problem may be caused by this root issue in REST API https://github.com/directus/directus/issues/23781
